frontend/src/views/ComplianceReports/components/AssessmentCard.jsx

```diff
@@ -200,20 +195,29 @@ export const AssessmentCard = ({
                       {t('report:supplementalWarning')}
                     </BCTypography>
                     <Box>
-                      <BCButton
-                        data-test="create-supplemental"
-                        size="small"
-                        variant="contained"
-                        color="primary"
-                        onClick={() => {
-                          createSupplementalReport()
-                        }}
-                        startIcon={<Assignment />}
-                        sx={{ mt: 2 }}
-                        disabled={isLoading}
+                      <Tooltip
+                        title={
+                          reportData.isNewest
+                            ? ''
+                            : 'Government has a reassessment in progress.'
```

Comment:
Just want to confirm that business area is ok with telling suppliers this is the case?

Reply:
yep!